src/server/actions/org.actions.ts

```diff
@@ -62,6 +64,55 @@ export const getReviewerPublicKeyAction = userAction(async () => {
 
 export type OrgUserReturn = Awaited<ReturnType<typeof getUsersForOrgAction>>[number]
 
+const updateOrgSettingsSchema = z
+    .object({
+        orgSlug: z.string(),
+    })
+    .merge(orgSchema.pick({ name: true, description: true }))
+
+export const updateOrgSettingsAction = orgAdminAction(async ({ orgSlug, name, description }) => {
+    // Fetch the current organization
+    const currentOrg = await db.selectFrom('org').select('name').where('slug', '=', orgSlug).executeTakeFirst()
+    if (!currentOrg) {
+        throw new ActionFailure({ org: `Organization with slug ${orgSlug} not found.` })
+    }
```

Comment:
the `orgAdminAction` wrapper ensures that the org is loaded and and stores it in the context.  example: https://github.com/safeinsights/management-app/blob/main/src/app/organization/%5BorgSlug%5D/admin/admin-users.actions.ts#L21